cluster/points_writer.go

```diff
@@ -261,6 +262,12 @@ func (w *PointsWriter) writeToShard(shard *meta.ShardInfo, database, retentionPo
 
 	for _, owner := range shard.Owners {
 		go func(shardID uint64, owner meta.ShardOwner, points []tsdb.Point) {
+			defer func() {
+				if r := recover(); r != nil {
+					w.statMap.Add(statWriteRecover, 1)
+					w.Logger.Printf("Recovered from write failure. Points:\n%v\n", points)
+				}
+			}()
```

Comment:
As I mentioned, I'm open to alternatives. Should we panic instead?

Logging all thousands of points a second, on the partner's node, doesn't seem feasible.